docs/source/_static/dpf_entry.html

```diff
@@ -0,0 +1,3407 @@
+<h3 class="h3-main">Open the ACT console scripting in Mechanical</h3>
+<p class="p-main" id="des_PackageOpenACT">The ACT console can be opened by clicking on "Automation"/"Scripting" menus in Mechanical.</p>
+<h3 class="h3-main">Import DPF and connect it to the current console</h3>
+<p class="p-main" id="des_PackageImportDPFACT">To import DPF�s capabilities into the ACT console, the user should import mech_dpf to access helpers and import the framework contained in the Ans.DataProcessing module. To access data from the current Mechanical windows, DPF�s helpers should be linked to it through the extAPI.</p>
+
+<h3 class="h3-main">DPF's helpers to access mechanical's data</h3>
+<p class="p-main">Diverse Mechanical�s and DPF's data can be accessed: </p>
+<ul class="ul-main">
+    <li  id="des_PackageACTDoc">this documentation can be generated via:</li>
+    <li id="des_PackageACTDataSources">the result files of Mechanical�s analysis: to collect the DataSources (DPF�s entity containing result file paths) the user can write:</li>
+    <li id="des_PackageACTMeshScoping">a mesh selection in the interface: to collect a mesh Scoping (DPF�s entity representing a list of ids of nodes or elements), once a geometry selection is picked out in the interface, it can be accessed via:</li>
+</ul>
+
+<h2 class="h2-main">Python Result</h2>
+<p class="p-main">The Python Result object allows to evaluate a workflow and to visualize its outputs in Mechanical (contours/graph/mesh...).</p>
+<h3 class="h3-main">Enable the Python Result</h3>
+<p class="p-main">To enable its use, the feature flags �Beta Options� (tab Tools/Appearance) and �Create Python Results� (tab Tools/Mechanical) must be checked in Workbench interface. From Mechanical, a Python Result object can be inserted under the Solution object using contextual menu or using Model ribbon menu.</p>
+<h3 class="h3-main">Evaluate a Python Result</h3>
+<p class="p-main">Once the python result object is added (see previous part) and written (a template script to calculate the Total Deformation result at the last step is provided by default) and the analysis is solved, an evaluation is needed .Any changes to the script�s contents will make the state of the Python Result object go �Undefined�. Users must then �Connect� the callback back to the event before evaluating by right clicking on the python result object and selecting "Connect". The user can finally click on "Solve" or "Evaluate Result".</p>
```

Comment:
Once the Python Result object is added (see previous part), written (a template script to calculate the Total Deformation result at the last step is provided by default,) and the analysis is solved, an evaluation is required. Any changes to the script's contents will cause the state of the Python Result object to change to _Undefined_.
To perform the evaluation:

1. Connect the callback to the event by right-clicking the Python Result object and selecting **Connect** from the context menu. 
2. Click **Solve** or **Evaluate Result**.

Reply:
updated in TFS